src/main/kotlin/apply/application/EvaluationTargetService.kt

```diff
@@ -0,0 +1,106 @@
+package apply.application
+
+import apply.domain.applicant.ApplicantRepository
+import apply.domain.applicationform.ApplicationFormRepository
+import apply.domain.cheater.CheaterRepository
+import apply.domain.evaluation.Evaluation
+import apply.domain.evaluation.EvaluationRepository
+import apply.domain.evaluationtarget.EvaluationTarget
+import apply.domain.evaluationtarget.EvaluationTargetRepository
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import javax.annotation.PostConstruct
+import javax.transaction.Transactional
+
+@Transactional
+@Service
+class EvaluationTargetService(
+    private val evaluationTargetRepository: EvaluationTargetRepository,
+    private val evaluationRepository: EvaluationRepository,
+    private val applicationFormRepository: ApplicationFormRepository,
+    private val applicantRepository: ApplicantRepository,
+    private val cheaterRepository: CheaterRepository
+) {
+    fun findByEvaluationId(evaluationId: Long): List<EvaluationTarget> =
+        evaluationTargetRepository.findByEvaluationId(evaluationId)
+
+    fun load(evaluationId: Long) {
+        val evaluation = evaluationRepository.findByIdOrNull(evaluationId) ?: throw IllegalArgumentException()
+        val cheaterApplicantIds = cheaterRepository.findAll().map { it.applicantId }
+        val isSavedBefore: Boolean = evaluationTargetRepository.existsByEvaluationId(evaluationId)
+
+        val updatingEvaluationTargets =
+            createUpdatingEvaluationTargets(evaluation).filter { !cheaterApplicantIds.contains(it.applicantId) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            createUpdatingEvaluationTargets(evaluation).filterNot { cheaterApplicantIds.contains(it.applicantId) }
```

Reply:
그쵸 코틀린에 없을 리가 없군요..😂

처리하겠슴니다.